Demo/Application/Base/AppDelegate.swift

```diff
@@ -64,7 +64,6 @@ import BraintreeCore
         if let preferences = settings.object(forKey: "PreferenceSpecifiers") as? Array<[String: Any]> {
             var defaultsToRegister: [String: Any] = [:]
             preferences.forEach { prefSpecification in
-                print(prefSpecification)
```

Comment:
Looks like some changes that were made on `main` are showing as diffs here and below, think we need to resolve those since they shouldn't show as a diff here.